packages/shopify-codemod/transforms/arguments-to-args-spread.js

```diff
@@ -0,0 +1,65 @@
+export default function argumentsToArgsSpread({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+  function isArglessIIFE(path) {
+    return j.match(path, {
+      type: j.CallExpression.name,
+      callee: {
+        type: j.ArrowFunctionExpression.name,
+        params: (params) => params.length === 0,
+      },
+      arguments: (args) => args.length === 0,
```

Comment:
I could be wrong, but I am not sure that it matters if the IIFE is passed any arguments.